addons/l10n_ec/models/account_chart_template.py

```diff
@@ -7,6 +7,45 @@
 class AccountChartTemplate(models.Model):
     _inherit = "account.chart.template"
 
+    def _load(self, company):
+        # EXTENDS account to setup taxes groups accounts configuration
+        res = super()._load(company)
+        self._l10n_ec_configure_ecuadorian_tax_groups_accounts(company)
+        return res
+
+    def _l10n_ec_configure_ecuadorian_tax_groups_accounts(self, companies):
+        '''
+        Set tax groups accounts for automatic closing entry in 103 and 104 reports
+        The structure of the variable with the list of accounts by tax group:
+        ('<tax_group_record_id>', '<payable_account_code>', '<receivable_account_code>')
+        '''
+        _TAX_GROUPS_ACCOUNTS_LIST = [
+            ('tax_group_vat_08', '21070102', '11050202'),
+            ('tax_group_vat_12', '21070102', '11050202'),
+            ('tax_group_vat14', '21070102', '11050202'),
+            ('tax_group_vat0', '21070102', '11050202'),
+            ('tax_group_vat_not_charged', '21070102', '11050202'),
+            ('tax_group_vat_exempt', '21070102', '11050202'),
+            ('tax_group_ice', '21070104', '21070104'),
+            ('tax_group_irbpnr', '21070105', '21070105'),
+            ('tax_group_withhold_vat_sale', '21070102', '11050203'),
+            ('tax_group_withhold_vat_purchase', '21070102', '11050203'),
+            ('tax_group_withhold_income_sale', '21070103', '11050201'),
+            ('tax_group_withhold_income_purchase', '21070103', '11050201'),
+            ('tax_group_outflows', '21070106', '11050205'),
+            ('tax_group_others', '21070106', '11050205'),
+        ]
+        for tax_group_xml_id, payable_account_code, receivable_account_code in _TAX_GROUPS_ACCOUNTS_LIST:
+            for company in companies.filtered(lambda company: company.account_fiscal_country_id.code == 'EC'):
```

Comment:
here you should check the Chart template instead because you do not know if the CoA is installed in that company